### DISCUSSION TITLE
When will "Package Runtime Config" aka DeploymentRuntimeConfig be available?

### DISCUSSION THREAD

**Discussion by Nold360** [crossplane/crossplane#4542] [General]
Hi,

i wanted to setup crossplane but the kyverno policies in my company are denying the provider deployments. 

I found a [proposal](https://github.com/crossplane/crossplane/blob/master/design/one-pager-package-runtime-config.md) for `DeploymentRuntimeConfig` but it's nowhere else mentioned. 

Can somebody tell me when this feature will be implemented?

**Comment by jbw976** [comment]
This feature was released as part of v1.14.0: https://github.com/crossplane/crossplane/releases/tag/v1.14.0

You can read about how to use it in the docs: https://docs.crossplane.io/latest/concepts/providers/#runtime-configuration

🎉
